blockchain-scrapers/docker-compose.yml

```diff
@@ -96,6 +96,25 @@ services:
       restart_policy:
         delay: 2s
         window: 20s
+  monero:
+    build:
```

Comment:
Now we're missing an image name:

    image:
      ${DOCKER_HUB_LOGIN}/monero